Makefile.dep

```diff
@@ -105,6 +105,13 @@ FEATURES_OPTIONAL += no_idle_thread
 ifneq (,$(filter libstdcpp,$(FEATURES_USED)))
   # Also use C++ if libstdc++ is used
   FEATURES_REQUIRED += cpp
+  USEMODULE += cpp
```

Comment:
As it already includes `FEATURES_REQUIRED += cpp` it should not be necessary to also include the module.

Reply:
ping